includes/class-wc-gateway-ppec-settings.php

```diff
@@ -306,7 +306,7 @@ public function get_paypal_locale() {
 		if ( ! in_array( $locale, $this->_supported_locales ) ) {
 			$locale = 'en_US';
 		}
-		return $locale;
+		return apply_filters( 'woocommerce_paypal_express_checkout_get_paypal_locale', $locale );
```

Comment:
I believe `get` is unnecessary in the name of the filter here – should be enough to filter the `paypal_locale` or just `locale` (with `woocommerce_paypal_express_checkout` as prefix).

Reply:
Good point!